package com.gravityer.authservice.exceptions;

public class InternalErrorException {
}
